<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="桂宝元" name="GiONEE_GN5001S " tests="2" failures="2" timestamp="13 Aug 2018 10:40:30 GMT" time="0.414" errors="0">
  <testcase name="f" time="0.362" classname="gby.testng.testcase.InitDriverCase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at gby.appium.ui.SetUpDriver.getDriver(SetUpDriver.java:40)
	at gby.testng.testcase.InitDriverCase.f(InitDriverCase.java:38)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="0.028" classname="gby.testng.testcase.LoginCase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at gby.testng.testcase.LoginCase.f(LoginCase.java:11)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- f -->
</testsuite> <!-- GiONEE_GN5001S  -->
